components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -213,6 +213,14 @@ be lost if SCREAM_HACK_XML is not enabled.
       <lambda_high type="real" doc="maximum value of stability correction.">0.04</lambda_high>
       <lambda_slope type="real" doc="slope of change from lambda_low to lambda_high.">2.65</lambda_slope>
       <lambda_thresh type="real" doc="stability threshold for which to apply more stability correction.">0.02</lambda_thresh>
+      <thl2tune type="real" doc="Temperature variance tuning factor">1.0</thl2tune>
+      <qw2tune type="real" doc="Moisture variance tuning factor">1.0</qw2tune>
+      <qwthl2tune type="real" doc="Temperature moisture covariance">1.0</qwthl2tune>
+      <w2tune type="real" doc="Vertical velocity variance">1.0</w2tune>
+      <length_fac type="real" doc="Length scale factor">0.5</length_fac>
+      <c_diag_3rd_mom type="real" doc="w3 factor">7.0</c_diag_3rd_mom>
```

Comment:
What is `w3` in this context?

Reply:
To be honest, I don't know.  This was the description text I got from @bogensch .  Peter, is there a better way to describe the variable `c_diag_3rd_mom`?

Reply:
Perhaps wording it as "Third moment vertical velocity damping factor" would be more descriptive.

Reply:
That's a much more understandable description!